roles/ceph-mds/tasks/docker/start_docker_mds.yml

```diff
@@ -1,4 +1,52 @@
 ---
+# Use systemd to manage container on Atomic host
+- name: generate systemd unit file
+  become: true
+  template:
+    src: "{{ role_path }}/templates/ceph-mds.service.j2"
+    dest: /var/lib/ceph/ceph-mds@.service
+    owner: "root"
+    group: "root"
+    mode: "0644"
+  when:
+    is_atomic or
+    ansible_os_family == 'CoreOS'
+
+- name: link systemd unit file for mds instance
+  file:
+    src: /var/lib/ceph/ceph-mds@.service
+    dest: /etc/systemd/system/multi-user.target.wants/ceph-mds@{{ ansible_hostname }}.service
+    state: link
+  when:
+    is_atomic or
+    ansible_os_family == 'CoreOS'
+
+- name: enable systemd unit file for mds instance
+  shell: systemctl enable /etc/systemd/system/multi-user.target.wants/ceph-mds@{{ ansible_hostname }}.service
+  failed_when: false
+  changed_when: false
+  when:
+    is_atomic or
+    ansible_os_family == 'CoreOS'
+
+- name: reload systemd unit files
+  shell: systemctl daemon-reload
```

Comment:
Made recommended change.

On Mon, Jun 6, 2016 at 3:09 AM, Leseb <notifications@github.com> wrote:

> In roles/ceph-mds/tasks/docker/start_docker_mds.yml
> <https://github.com/ceph/ceph-ansible/pull/840#discussion_r65866389>:
>
> > +    dest: /etc/systemd/system/multi-user.target.wants/ceph-mds@{{ ansible_hostname }}.service
> > +    state: link
> > +  when:
> > +    is_atomic or
> > +    ansible_os_family == 'CoreOS'
> > +
> > +- name: enable systemd unit file for mds instance
> > +  shell: systemctl enable /etc/systemd/system/multi-user.target.wants/ceph-mds@{{ ansible_hostname }}.service
> > +  failed_when: false
> > +  changed_when: false
> > +  when:
> > +    is_atomic or
> > +    ansible_os_family == 'CoreOS'
> > +
> > +- name: reload systemd unit files
> > +  shell: systemctl daemon-reload
>
> ditto
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ceph/ceph-ansible/pull/840/files/e047b0a8ed803f1a73f7caebe5c3da4a85c03ae7#r65866389>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/APWGwJiWJVg9vjcTGFY_6_UY92DTIK0Vks5qI_HAgaJpZM4ItD49>
> .
>



-- 
Jim Curtis
Principal Software Engineer
Emerging Technologies
Red Hat Inc.

jim.curtis@redhat.com